app/models/node.rb

```diff
@@ -37,7 +35,7 @@ class Node < ApplicationRecord
 
   scope :active, -> { where(deactivated_by: nil) }
 
-  scope :with_cost_code, -> { where(Node.arel_table[:cost_code].matches('S%')) }
+  scope :with_cost_code, -> { where.not(cost_code: nil) }
```

Comment:
Doesn't this allow any cost code now?

Reply:
No, this is for retrieving Nodes with cost codes from the database. Cost code format and general validation is handled when that data is added. Previously, this was essentially validating data on its way out of the database as well [@jbeast 2018]